MAINTAIN.md

```diff
@@ -71,17 +71,36 @@ inform users of the change.
 
 In general, when a feature is slated to be removed it should:
 
-1. Be marked deprecated in the _next_ release
-  - This includes a note in the release notes (include a "Deprecation Warning"
-    section just below "Breaking Changes")
+1. Be _soft_ deprecated in the _next_ release
+  - Use of the deprecated feature will still work.
+  - This means deprecating via documentation only.
+  - Must include a note in `news.txt` under `DEPRECATIONS`.
+  - Lua features can be annotated with `@deprecated`.
+2. Be _hard_ deprecated in a following a release in which it was soft deprecated.
+  - Use of the deprecated feature will still work but may issue a warning.
   - Lua features can use `vim.deprecate()`
   - Features implemented in Vimscript or in C will need bespoke implementations
     to communicate to users that the feature is deprecated
-2. Be removed in a release following the release in which it was marked
-   deprecated
+3. Be removed in a release following the release in which it was hard deprecated
```

Comment:
If possible it's probably preferable to `vim.deprecate()` with error/stacktrace?

Reply:
I don't think `vim.deprecate` can currently facilitate this (...yet).